### DISCUSSION TITLE
Unable to resolve path to module 'react-day-picker/style.module.css' in v9.0.8

### DISCUSSION THREAD

**Discussion by shareef-dweikat** [gpbl/react-day-picker#2431] [Support]
Hi

## Code 

```tsx
import {
  DayPicker,
  DateRange,
  ClassNames,
  DayPickerRangeProps,
  DayPickerSingleProps,
} from 'react-day-picker';
import classNames from 'react-day-picker/style.module.css';
```

## Expected Behavior
The style import to work fine

## Actual Behavior
I get this error "Unable to resolve path to module 'react-day-picker/style.module.css'.eslint[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.29.1/docs/rules/no-unresolved.md)
Cannot find module 'react-day-picker/style.module.css' or its corresponding type declarations." 

Version: "react-day-picker": "^9.0.8"
I have tried to reinstall node_modules, it did not work. I am using **Gatsby**.

**Comment by gpbl** [comment]
@shareef-dweikat any help replicating this?

**Comment by sshmyg** [comment]
@shareef-dweikat Try to import styles from `src`, like `react-day-picker/src/style.module.css`.

I have like this in my codebase `import { default as defaultStyles } from 'react-day-picker/dist/style.module.css';` and it works, not sure why, probably becasue of mapping in `package.json`.
